e2e/steps_file.js

```diff
@@ -148,6 +176,13 @@ module.exports = function () {
       await this.click('Continue');
     },
 
+    async addAnotherElementToCollection() {
+      const numberOfElements = await this.grabNumberOfVisibleElements('.collection-title');
+      this.click('Add new');
+      this.waitNumberOfVisibleElements('.collection-title', numberOfElements + 1);
+      this.wait(0.5); // add extra time to allow slower browsers to render all fields (just extra precaution)
```

Comment:
Not a fan of this kind of waits. We should use methods that specify what exactly we are waiting for. Also, how do we know that 0.5 seconds will be sufficient? Feels like slowing the test down without having actual confidence in its success.

Reply:
will just remove the wait